api/DocumentFragment.json

```diff
@@ -53,13 +53,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentFragment/DocumentFragment",
           "support": {
             "chrome": {
-              "version_added": "28"
+              "version_added": "29"
```

Comment:
I can confirm this by testing http://mdn-bcd-collector.appspot.com/tests/api/DocumentFragment/DocumentFragment on Chrome 28 and 29.